### PR TITLE
17 create messages model

### PR DESCRIPTION
**Pull request by ahmedirheem**
relates #17

### CODE REVIEW

**Review by Saeed99Madi — APPROVED** [review]
good job

File: server/models/messages.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from '../database/config/connection';
+
+class MessageDatatypes extends Model {
+  declare id: number;
```
Comment (alaaahmade) [inline server/models/messages.ts:5]: remove id column
id column will be added by default on sequelize

File: server/models/messages.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  declare player_id: number;
+  declare match_id: number;
+  declare message: string;
+}
+
+export const Message = MessageDatatypes.init(
+  {
+    username: { type: DataTypes.STRING(50), allowNull: false },
+    id: {
```
Comment (alaaahmade) [inline server/models/messages.ts:14]: remove id column
id column will be added by default on sequelize

File: server/models/messages.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+      allowNull: false,
+    },
+    player_id: {
```
Comment (alaaahmade) [inline server/models/messages.ts:20]: player_id should have references to be Foreign Key

like :  player_id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      references: {
        model: 'Player',
        key: 'id',
      },

File: server/models/messages.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
```
Comment (alaaahmade) [inline server/models/messages.ts:22]: remove autoIncrement from player_id because it's a foreign key

File: server/models/messages.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      allowNull: false,
+    },
+    match_id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
```
Comment (alaaahmade) [inline server/models/messages.ts:27]: remove autoIncrement from player_id because it's a foreign key

**Review by alaaahmade — CHANGES_REQUESTED** [review]

**Comment by Saeed99Madi** [comment]
No Description Provided !!

**Review by Saeed99Madi — APPROVED** [review]
Great Job Ahmed @ahmedirheem 
:100:

**Review by alaaahmad123 — APPROVED** [review]

**Review by alaaahmade — APPROVED** [review]

**Review by Heba-Adwan — APPROVED** [review]

**Pull request merged**